### PR TITLE
PS-216 Add v2.0_large-signals-demo and v1.8 prefix to other examples

### PR DESCRIPTION
**Pull request by jinlinGuan**

### CODE REVIEW

File: v2.0_large-signals-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+```shell
+# ./build.sh <push_or_not> <image tag name> 
+cd path-to-setup
+./build.sh true large-signals-demo
+```
+
+## Run the Demo
+> **⚠ Must have already logged in Edge Builder and added the license.**
+
+> **⚠ Replace BrokerAddress, clientcert, and clientke in the app-service/app-aws-modbus.toml and app-servie/app-aws-opc-ua.toml with a valid AWS IOT config**
```
Comment (cloudxxx8) [inline v2.0_large-signals-demo/README.md]: typo, clientkey

File: v2.0_large-signals-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+### Remove app, appConfig, appConfigFile, appDefinition and node
+Run the command to remove app:
+```shell
+edgebuilder-cli app rm -a <app-name or app-id>
+```
+Run the command to remove app config:
+```shell
+# edgebuilder-cli appConfig rm -c <app-config-name or app-config-id> 
+e.g.
```
Comment (cloudxxx8) [inline v2.0_large-signals-demo/README.md]: miss the e.g. content here

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: v2.0_large-signals-demo/README.md (hunk 1 above)
Comment (jinlinGuan) [reply v2.0_large-signals-demo/README.md]: fixed

File: v2.0_large-signals-demo/README.md (hunk 2 above)
Comment (jinlinGuan) [reply v2.0_large-signals-demo/README.md]: added

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**